src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonApiClient.kt

```diff
@@ -20,7 +20,7 @@ class PrisonApiClient(@Qualifier("oauthPrisonClient") val prisonerApiWebClient:
   fun getHdcStatus(bookingId: Long): Mono<PrisonerHdcStatus> {
     return prisonerApiWebClient
       .get()
-      .uri("/offender-sentences/booking/$bookingId/home-detention-curfews/latest")
+      .uri("/offender-sentences/booking/{bookingId}/home-detention-curfews/latest", bookingId)
```

Comment:
this is addressing a minor memory leak - each unique url creates an instance of a long running micrometer Tag classes - so one for ever booking id that is passed here! 

But using parameterised routes this avoids that. 
For more details there's a [slack thread](https://mojdt.slack.com/archives/C69NWE339/p1697456808633699)